src/OpenTelemetry.ResourceDetectors.Azure/README.md

```diff
@@ -31,3 +32,50 @@ var tracerProvider = Sdk.CreateTracerProviderBuilder()
                             .AddDetector(new AppServiceResourceDetector()))
                         .Build();
 ```
+
+| Attributes recorded by AppServiceResourceDetector |
+|---------------------------------------------------|
+| azure.app.service.stamp                           |
+| cloud.provider                                    |
+| cloud.platform                                    |
+| cloud.resource_id                                 |
+| cloud.region                                      |
+| deployment.environment                            |
+| host.id                                           |
+| service.instance.id                               |
+| service.name                                      |
+
+## VM Resource Detector
+
+Adds resource attributes for the applications running in an Azure virtual machine.
+The following example shows how to add `AzureVMResourceDetector` to
+`TracerProvider` configuration, but this can be added to logs and metrics
+as well.
+
+```csharp
+using OpenTelemetry;
+using OpenTelemetry.ResourceDetectors.Azure;
+using OpenTelemetry.Resources;
+
+var tracerProvider = Sdk.CreateTracerProviderBuilder()
+                        // other configurations
+                        .SetResourceBuilder(ResourceBuilder
```

Comment:
Use `ConfigureResource` instead.

`.ConfigureResource(resource=> resource.AddDetector(new AzureVMResourceDetector()))`

Reply:
@TimothyMothra Something to be considered for a follow-up PR.